cast/java/src/testFixtures/java/com/ibm/wala/cast/java/test/IRTests.java

```diff
@@ -492,12 +495,12 @@ public static void populateScope(
         }
       }
     }
-    assert foundLib : "couldn't find library file from " + libs;
+    assertThat(foundLib).withFailMessage(() -> "couldn't find library file from " + libs).isTrue();
 
     for (String srcFilePath : sources) {
       String srcFileName = srcFilePath.substring(srcFilePath.lastIndexOf(File.separator) + 1);
       File f = new File(srcFilePath);
-      assertTrue(f.exists(), "couldn't find " + srcFilePath);
+      assertThat(f).withFailMessage(() -> "couldn't find " + srcFilePath).exists();
```

Comment:
I'm really finding the readability to be reduced with all these `withFailMessage()` calls.  They kind of defeat the whole readability advantage of the fluent interface.  I'm leaning towards just getting rid of them and adding back custom messages where we need them, or possibly splitting tests later if they are asserting too many things.

Reply:
d333b13108d4f1ca7bd256d4f97bbab67dcec8ba removes all `withFailMessage` calls.